Uchu.StandardScripts/AvantGardens/BusDoor.cs

```diff
@@ -0,0 +1,58 @@
+using System.Numerics;
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.AvantGardens
+{
+    [ZoneSpecific(1100)]
+    public class BusDoor : NativeScript
+    {
+        public override Task LoadAsync()
+        {
+            var gameObjects = HasLuaScript("l_ag_bus_door.lua");
+
+            foreach (var gameObject in gameObjects)
+            {
+                if (!gameObject.TryGetComponent<MovingPlatformComponent>(out var movingPlatformComponent)) continue;
+                movingPlatformComponent.Stop();
+                
+                bool doorOpen = default;
+                Listen(Zone.OnTick, () =>
```

Comment:
Instead of doing this OnTick, I'd recommend trying to do this OnColission with bounding box physics component. Imo this is preferred over the polling method shown here. To easily discover the proper physics item to collide with you can add debugging statements to the respective methods in the player class that trigger when a collision happens.